lab-integration/src/main.ts

```diff
@@ -0,0 +1,42 @@
+import { DateTime } from 'luxon';
+
+import { promises as fs, watch } from "fs";
+import path from "path";
+import { TestsResult } from "./types";
+
+const filePath = path.join(process.cwd(), "../test-results.json");
+
+(async () => {
+  console.info("SCRIPT IS RUNNING...");
+
+  let lastTestResultDate = DateTime.now();
+
+  watch(filePath, async (eventType) => {
+    const stringifiedDataset = await fs.readFile(filePath, "utf8");
+    const parsedDataset = JSON.parse(stringifiedDataset) as TestsResult[];
+    const dataset = parsedDataset.map((testResult) => {
+      const Data = DateTime.fromFormat(testResult.Data, 'dd/LL/yyyy, HH:mm:ss', { locale: 'pt-BR' });
+
+      return {
+        ...testResult,
+      Data: Data.isValid ? Data : DateTime.now()
```

Comment:
Instead of setting it to the current date and time, can we filter based on `Data.isValid`? If `Data` is invalid, we can't do anything about that.